CenterForOpenScience/ember-osf-preprints
# [IN-97][Preprints] Get more data from preprint model

## Purpose

Grab more data from the preprint model instead of the node

## Summary of Changes
- Update the submit controller to save more data (title, tags, description) on the preprint model
- Update content controller and template to retrieve more data from preprint model
- Replace terminologies or variable names to reflect the changes.
- Update tests

## Side Effects / Testing Notes
The changes in this ticket affect preprint `submit` and`detail` pages.
- On the preprint submit page:
- At each section, make sure to fill out the info including the non-required fields: `tags, description, title` should not be blank to test this ticket.
- Make sure each section saves without errors (developer console or errors growlers)
- Visit the preprint you just created, and make sure there are no errors and
all the preprint info you entered is available, especially the `tags, title, description`.

## Ticket

[IN-97](https://openscience.atlassian.net/browse/IN-97)

# Reviewer Checklist
